testing/test_config.py

```diff
@@ -1264,15 +1264,16 @@ def pytest_load_initial_conftests(self):
     m = My()
     pm.register(m)
     hc = pm.hook.pytest_load_initial_conftests
-    values = hc._nonwrappers + hc._wrappers
-    expected = [
+    assert [x.function.__module__ for x in hc._nonwrappers] == [
```

Comment:
Decided to also split checking wrappers and non-wrappers, seems better to explicitly test them separately.